test/e2e/storage/drivers/in_tree.go

```diff
@@ -472,7 +472,7 @@ func newISCSIServer(cs clientset.Interface, namespace string) (config volume.Tes
 	}
 	pod, ip = volume.CreateStorageServer(cs, config)
 	// Make sure the client runs on the same node as server so we don't need to open any firewalls.
-	config.ClientNodeName = pod.Spec.NodeName
+	config.NodeSelector = map[string]string{v1.LabelHostname: pod.Spec.NodeName}
```

Comment:
@jsafrane I don't really understand this iscsi restriction. Is this still needed?

Reply:
Clouds tend to block iscsi (and nfs) in between nodes. At least in our downstream CI.